src/totalOrder.ml

```diff
@@ -1,14 +1,10 @@
-type comparison =
+(** A type represent a total order relationship *)
```

Comment:
In general, in imm.re i've avoid abbreviated names, hence I've used Ordering, Sequence, etc. In general i feel this optimizes for readability over terseness.